ansible/roles_ocp_workloads/ocp4_workload_rhtr_xraylab/templates/storage/subscription.yaml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ocs-operator
+  namespace: {{ ocp4_workload_rhtr_xraylab_ocs_namespace }}
+spec:
+  channel: {{ ocp4_workload_rhtr_xraylab_ocs_channel }}
+  installPlanApproval: Automatic
+  name: ocs-operator
+  source: redhat-operators
```

Comment:
Same here - should support Catalog Snapshots

Reply:
Will do

Reply:
Done!